cmake_downloader.py

```diff
@@ -38,13 +39,14 @@ def get_tarball_urls() -> List[str]:
     return result
 
 
-def download_and_extract(url: str, path: str):
+def download_and_extract(url: str, path: Union[str, Path], clobber: bool = False) -> None:
     # derive file directory name from URL
     file_name_start_pos = url.rfind('/') + 1
     file_name = url[file_name_start_pos:]
     file_wo_ext = file_name.replace('.tar.gz', '').replace('.zip', '')
 
-    if not os.path.exists(os.path.join(path, file_wo_ext)):
+    if clobber or not os.path.exists(os.path.join(path, file_wo_ext)):
```

Comment:
This seems to force a download if `clobber == True` even if the file is present.